.travis.yml

```diff
@@ -7,44 +7,12 @@ notifications:
 services:
   - docker
-matrix:
-  include:
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=true  COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=5.1.0
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=master
-      if: type != pull_request
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=master
-      if: type != pull_request
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=master
-      if: type != pull_request
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=master
-      if: type != pull_request
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=develop
-      if: type != pull_request
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=develop
-      if: type != pull_request
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=develop
-      if: type != pull_request
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=develop
-      if: type != pull_request
-  fast_finish: true
-  allow_failures:
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=master
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=master
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=master
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=master
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=develop
-    - env: UBUNTU_VERSION=16.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=develop
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=gcc   HDF5_VERSION=1.10.4 MOAB_VERSION=develop
-    - env: UBUNTU_VERSION=18.04 HOUSEKEEPING_ONLY=false COMPILER=clang HDF5_VERSION=1.10.4 MOAB_VERSION=develop
+
 before_install:
   # Set some local variables
   - docker_image=svalinn/dagmc-ci-ubuntu-${UBUNTU_VERSION}
   - dagmc_build_dir=/root/build/${COMPILER}/DAGMC-moab-${MOAB_VERSION}-hdf5-${HDF5_VERSION}
-  - script_dir=${dagmc_build_dir}/DAGMC/docker
+  - script_dir=${dagmc_build_dir}/DAGMC/scripts
```

Comment:
I don't think I understand why the scripts need to be in a separate directory from the Dockerfile. All of the scripts in this directory are CI-specific scripts that are only intended to be used in conjunction with the docker image.

Reply:
I think I got confuse because some of the file where `travis.xxx` and some just `build` I though that the build one could be used elsewhere.
@ljacobson64 What about we put all of them in a CI folder as you suggested (including the `Dockerfile`) and add a `docker.` prefix in the build scripts that are intended to be used in the docker building process.

last question where do we use the `build_dagmc.sh` script  ?

Reply:
I like that idea. So there'd be a folder called `CI` and it would contain these files

```
docker.build.dagmc.sh
docker.build.geant4.sh
docker.build.hdf5.sh
docker.build.moab.sh
docker.env.sh
Dockerfile
travis.housekeeping.sh
travis.install.sh
travis.tests.sh
update_docker.sh
```

Reply:
build_dagmc is only used in travis, so I named it `travis.`
still working on making sure update_docker still work before pushing it

Reply:
`env.sh` is used in both, so I kept it without a prefix

Reply:
Sorry to be late here, but why `CI/travis.xxx` and `CI/docker.xxxx` and not `CI/travis/xxxx` and `CI/docker/xxxx`?